client/my-sites/stats/annual-highlights-section/index.tsx

```diff
@@ -1,4 +1,4 @@
-import { AnnualHighlightCards } from '@automattic/components';
+import AnnualHighlightCards from '@automattic/components/src/highlight-cards/annual-highlight-cards';
```

Comment:
I'm curious about this change to direct imports. I see it was changed back in a later commit. Was this just to keep the build system happy while the renaming that was happening?

Reply:
I wanted to verify if our build system was properly tree-shaking unused dependencies for each entry point; I was concerned that exporting additional components in `src/index.tsx` may bloat all bundles importing the components package.

Luckily, our build system does properly tree-shake unused dependencies. I added them back to `index.tsx` to improve the DevUX for consuming these components.

Reply:
Nice! Thanks for the clarification!